src/unpack.rs

```diff
@@ -63,8 +63,21 @@ pub async fn unpack(options: UnpackOptions) -> Result<()> {
 
     validate_metadata_file(unpack_dir.join(PIXI_PACK_METADATA_PATH)).await?;
 
-    let target_prefix = options.output_directory.join(options.env_name);
+    // HACK: The `Installer` created below, as well as some code in building
+    // packages from source will utilize rayon for parallelism. By using rayon
+    // it will implicitly initialize a global thread pool. However, uv
+    // has a mechanism to initialize rayon itself, which will crash if the global
+    // thread pool was already initialized. To prevent this, we force uv the
+    // initialize the rayon global thread pool, this ensures that any rayon code
+    // that is run will use the same thread pool.
+    //
+    // One downside of this approach is that perhaps it turns out that we won't need
+    // the thread pool at all (because no changes needed to happen for instance).
+    // There is a little bit of overhead when that happens, but I don't see another
+    // way around that.
+    LazyLock::force(&RAYON_INITIALIZE);
```

Comment:
thanks a lot on your help with this @tdejager!
xref https://github.com/prefix-dev/pixi/blob/4dc02c840d63e75f16a2da6a8fc74a7f67218cb3/src/environment/conda_prefix.rs#L294

Reply:
also xref https://github.com/rayon-rs/rayon/issues/93